website/docs/r/api_management_logger.html.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_api_management_logger"
+sidebar_current: "docs-azurerm-resource-api-management-logger"
+description: |-
+  Manages an API Management Logger.
+---
+
+# azurerm_api_management_logger
+
+Manages an API Management Logger.
+
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West US"
+}
+
+resource "azurerm_application_insights" "example" {
+  name                = "example-appinsights"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  application_type    = "Other"
+}
+
+resource "azurerm_api_management" "example" {
+  name                = "example-apim"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  publisher_name      = "My Company"
+  publisher_email     = "company@terraform.io"
+
+  sku {
+    name     = "Developer"
+    capacity = 1
+  }
+}
+
+resource "azurerm_api_management_logger" "example" {
+  name                = "example-logger"
+  api_management_name = "${azurerm_api_management.example.name}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  description         = "This logger is created by Terraform."
```

Comment:
since `description` is optional could we remove it from this example? we've [got a TODO](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/examples/todo.txt#L9) to add some additional examples for API Management once the other resources have been added